specs/whisk/beacon-chain.md

```diff
@@ -109,18 +109,8 @@ def IsValidWhiskOpeningProof(tracker: WhiskTracker, k_commitment: BLSG1Point, tr
 
 ```python
 class WhiskTracker(Container):
-    r_G: BLSG1Point  # r * G
-    k_r_G: BLSG1Point  # k * r * G
-```
-
-### `Validator`
-
-```python
-class Validator(bellatrix.Validator):
-    # ...
-    # Whisk
-    whisk_tracker: WhiskTracker  # Whisk tracker (r * G, k * r * G) [New in Whisk]
-    whisk_k_commitment: BLSG1Point  # Whisk k commitment k * BLS_G1_GENERATOR [New in Whisk]
+    r_g: BLSG1Point  # r * G
+    k_r_g: BLSG1Point  # k * r * G
```

Comment:
- See comment on main PR for keeping the user facing case strict snake https://github.com/ethereum/consensus-specs/pull/3205#issuecomment-1475260005